playbooks/roles/ocp-config/README.md

```diff
@@ -11,32 +11,49 @@ Requirements
+| chronyconfig.content        | no       | ""                                    | List of time NTP servers and options pair (see chronyconfig examples). If empty, bastion will try sync with some default ntp server (internet) AND local HW clock (with higher stratum). |
+| chronyconfig.allow          | no       | ""                                    | List of network cidr (X.X.X.X/Y) allowed to sync with bastion configured as NTP server |
+| dhcp_shared_network         | no       |                                       | Flag to update DHCP server work on a shared network. (Neither ACK nor NACK unknown clients) |
+| cni_network_provider        | no       | OVNKubernetes                         | Sets the default Container Network Interface (CNI) network provider for the cluster |
+| cni_network_mtu             | no       |                                       | MTU value to assign to the CNI network. Recommended values for OpenshiftSDN: <NIC MTU> - 50; OVNKubernetes: <NIC MTU> - 100 |
+| cluster_network_cidr        | no       | 10.128.0.0/14                         | Network (in CIDR) used for the pod networks.
+| cluster_network_hostprefix  | no       | 23                                    | The subnet prefix length to assign to each individual node. (netmask in CIDR format)
+| service_network             | no       | 172.30.0.0/16                         | Network (in CIDR) used for the service network.
+| rhcos_pre_kernel_options    | no       | []                                    | List of day-1 kernel options for RHCOS nodes eg: ["rd.multipath=default","root=/dev/disk/by-label/dm-mpath-root"] |
+| luks_compliant.enabled      | yes      | false                                 | Set it true if you prefer to enable LUKS in ocp deployment |
+| luks_config                 | yes      | []                                    | List of tang servers and thumbprint to apply |
+| luks_filesystem_device      | yes      | /dev/mapper/root                      | Set the Path of device to be luks encrypted |
+| luks_format                 | yes      | xfs                                   | Set the Format of the FileSystem to be luks encrypted |
+| luks_wipeFileSystem         | yes      | true                                  | Configures the FileSystem to be wiped |
+| luks_device                 | yes      | /dev/disk/by-partlabel/root           | Set the Path of luks encrypted partition |
+| luks_label                  | yes      | luks-root                             | Set the value for user label of luks encrpted partition |
+| luks_options                | yes      | ["--cipher", "aes-cbc-essiv:sha256"]  | Set List of luks options for the luks encryption |
+| luks_wipeVolume             | yes      | true                                  | Configures the luks encrypted partition to be wiped |
+| luks_name                   | yes      | root                                  | Set the value for the user label of Filesystem to be luks encrypted |
```

Comment:
Added changes to column width in order to have clean view of variable list

Reply:
@prb112 Can you try resolving this conversation? as the message in PR shows as below:
![image](https://user-images.githubusercontent.com/63038004/197114734-fed819a2-fe7d-47c1-b782-1c69d2239e9e.png)